src/recon_buildblock/PoissonLogLikelihoodWithLinearModelForMeanAndListModeDataWithProjMatrixByBin.cxx

```diff
@@ -532,6 +532,28 @@ compute_sub_gradient_without_penalty_plus_sensitivity(TargetT& gradient,
         }
     }
     info(boost::format("Number of used events: %1%") % num_used_events);
+
+  if (!add_sensitivity)
+    {
+      if (!this->get_use_subset_sensitivities())
+        error("PoissonLogLikelihoodWithLinearModelForMeanAndListModeDataWithProjMatrixByBin::"
+              "actual_compute_sub_gradient_without_penalty(): cannot subtract subset sensitivity because "
+              "use_subset_sensitivities is false.");
```

Comment:
This is not clean but I feel it is nessisery to prevent improper use of sensitivity subtraction.

Reply:
Not outdated